Source/TaggingCriteria/ChiExtractionTaggingCriterion.hpp

```diff
@@ -41,12 +40,17 @@ class ChiExtractionTaggingCriterion
 
     // The constructor
     ChiExtractionTaggingCriterion(const double dx, const int a_level,
-                                  const int a_max_level,
                                   const SphericalExtraction::params_t a_params,
                                   const bool activate_extraction = false)
         : m_dx(dx), m_deriv(dx), m_params(a_params), m_level(a_level),
-          m_max_level(a_max_level),
           m_activate_extraction(activate_extraction){};
+    // below is a constructor for backward compatibility
+    ChiExtractionTaggingCriterion(const double dx, const int a_level,
+                                  const int a_max_level,
+                                  const SphericalExtraction::params_t a_params,
+                                  const bool activate_extraction = false)
+        : ChiExtractionTaggingCriterion(dx, a_level, a_params,
+                                        activate_extraction){};
```

Comment:
I was going to complain at you for breaking backwards compatibility until I saw this. Well done! :stuck_out_tongue: